lib/src/data/schedule/schedule.dart

```diff
@@ -28,14 +28,17 @@ class Schedule {
 
 	/// The name of this schedule. 
 	final String name;
-	
+
+	/// A Map of weekdays to their default schedule
+	static late Map<String, Schedule> defaults;
```

Comment:
You can do this in three ways: 
1. `static late`, which means there is no value until it's set
2. `Map<String, Schedule>?`, which means it's null until set
3. `defaults = {};`, which means it's empty until set. 

While 3 is easier to work with, I think you should do 1 or 2, since it implies it's invalid to search up a default schedule before the data is loaded. You can follow my example and use `late`, which implies there should always be _some_ data and not having data is an error, but I think you should use a nullable map since `late` is kinda hacky in Dart but null is much more "normal".